src/BedethequeScraper2.py

```diff
@@ -2778,343 +2490,387 @@ def __init__(self):
+        self._ButtonCheckNone.Name = "ButtonCheckNone"
+        self._ButtonCheckNone.Size = System.Drawing.Size(120, 20)
+        self._ButtonCheckNone.TabIndex = 53
+        self._ButtonCheckNone.Text = Trans(94)
+        self._ButtonCheckNone.UseVisualStyleBackColor = False
+        self._ButtonCheckNone.Tag = "Button"
+        self._ButtonCheckNone.Click += self.button_Click
+        #
+        # ButtonCheckAll
+        #
+        self._ButtonCheckAll.BackColor = System.Drawing.Color.FromArgb(255, 192, 128)
+        self._ButtonCheckAll.Font = System.Drawing.Font("Microsoft Sans Serif", 8.25, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, 0)
+        self._ButtonCheckAll.Location = System.Drawing.Point(132, 308)
+        self._ButtonCheckAll.Name = "ButtonCheckAll"
+        self._ButtonCheckAll.Size = System.Drawing.Size(120, 20)
+        self._ButtonCheckAll.TabIndex = 52
+        self._ButtonCheckAll.Text = Trans(117)
+        self._ButtonCheckAll.UseVisualStyleBackColor = False
+        self._ButtonCheckAll.Tag = "Button"
+        self._ButtonCheckAll.Click += self.button_Click
+        #
+        # labelVersion
+        #
+        self._labelVersion.Font = System.Drawing.Font("Microsoft Sans Serif", 6.75, System.Drawing.FontStyle.Italic, System.Drawing.GraphicsUnit.Point, 0)
+        self._labelVersion.ImageAlign = System.Drawing.ContentAlignment.BottomCenter
+        self._labelVersion.Location = System.Drawing.Point(162, 380)
+        self._labelVersion.Name = "labelVersion"
+        self._labelVersion.Size = System.Drawing.Size(264, 16)
+        self._labelVersion.TabIndex = 19
+        self._labelVersion.Text = "Version " + VERSION + " (c) 2021 kiwi13 && maforget"
+        self._labelVersion.TextAlign = System.Drawing.ContentAlignment.BottomCenter
+        #
+        # labelDBGLOGMAX
+        #
+        self._labelDBGLOGMAX.Font = System.Drawing.Font("Microsoft Sans Serif", 8.25, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, 0)
+        self._labelDBGLOGMAX.ImageAlign = System.Drawing.ContentAlignment.MiddleLeft
+        self._labelDBGLOGMAX.Location = System.Drawing.Point(8, 124)
+        self._labelDBGLOGMAX.Name = "labelDBGLOGMAX"
+        self._labelDBGLOGMAX.Size = System.Drawing.Size(270, 21)
+        self._labelDBGLOGMAX.TabIndex = 16
+        self._labelDBGLOGMAX.Text = Trans(69)
         #
         # DBGLOGMAX
         #
-        self._DBGLOGMAX.Location = System.Drawing.Point(8, 120)
+        self._DBGLOGMAX.Location = System.Drawing.Point(300, 122)
         self._DBGLOGMAX.Name = "DBGLOGMAX"
-        self._DBGLOGMAX.Size = System.Drawing.Size(72, 23)
+        self._DBGLOGMAX.Size = System.Drawing.Size(40, 23)
         self._DBGLOGMAX.TabIndex = 4
-        self._DBGLOGMAX.Text = str(DBGLOGMAX)
+        self._DBGLOGMAX.TextAlign = HorizontalAlignment.Center
+        self._DBGLOGMAX.Text = str(int(DBGLOGMAX / 1024 / 1024))

```

Comment:
This is a breaking change. The default value is 10 000 bytes, so around 10KiB. So your default value will be 0.009 Mb, so since it's an int it will changed to 0. So on opening the setting window it will save the value to 0, So on each and every run it will ask to replace the files. 

We can't change the default value, because existing settings wouldn't not work either. 
 * We can either change it to a double/float
 * Just change it to Kb instead
 * But a floor to be 1 minimum, so it would be 10KB if you have an existing setting file, and at the same time change the minimum to at least 1MB.

Reply:
I choose to use floating value representing MegaBytes in UI and then store Bytes to fix that.